src/components/page-layout/page-layout.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Sticky from 'react-stickynode';
+import debounce from 'debounce';
+
+class PageLayout extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      height: 0
+    };
+    this.debounceHandleWindowResize = debounce(() => {
+      const height = document.body.clientHeight;
+      this.setState({
+        bottomBoundaryValue: height - 450
+      });
+    }, 200);
+  }
+
+  componentDidMount() {
+    this.debounceHandleWindowResize();
+    window.addEventListener('resize', this.debounceHandleWindowResize);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.debounceHandleWindowResize);
+  }
+
+  render() {
+    const { props, state } = this;
+    return (
+      <div className="grid">
+        <div
+          className={`col col--4-mm col--12 ${
+            props.sidebarColor
+          } px0 mt0-mm mt60`}
+        >
+          <Sticky
+            enabled={true}
+            bottomBoundary={state.bottomBoundaryValue}
+            innerZ={1}
+            top={props.sidebarContentStickyTop}
+            activeClass="bg-gray-faint shadow-darken-10"
+          >
+            <div className="txt-ms pt24-mm pt0 viewport-almost-mm scroll-auto ml36 mt-neg60 mt0-mm">
+              <div className="txt-l color-blue txt-fancy mb12 block-mm none">
+                {props.sidebarTitle}
+              </div>
```

Comment:
If a `sidebarTitle` is not provided you'll end up with a 12px margin beneath nothing — so, just some weird empty space. 

So maybe `sidebarTitle` should be required or else we only render that styled wrapper div if it's provided?